docs/technical-documentation/migrate-7x.md

```diff
@@ -0,0 +1,3 @@
+<iframe width="560" height="315" src="https://www.youtube.com/embed/fEDzfSPjKEo" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Consider using Youtube's "no cookie" URL: https://www.youtube-nocookie.com/embed/fEDzfSPjKEo

Reply:
This was my mistake I forgot to test this by actually building the Mkdocs. This works as is.